OloEngine/src/OloEngine/Networking/Lockstep/StateHash.cpp

```
@@ -0,0 +1,280 @@
+        0x5B6C0CFC,
+        0xCB6AABED,
+        0xBC6D9B7B,
+        0x256649C1,
+        0x52617957,
+        0xCC052DF4,
+        0xBB020362,
+        0x240B52D8,
+        0x5308224E,
+    };
+
+    u32 StateHash::Compute(const u8* data, u32 size)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)